.circleci/config.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           name: run tests
```

Comment:
Please put (move) uploading artifacts stage after test execution, but not before

Reply:
Done. Please check.